atomos/atomic.py

```diff
@@ -172,7 +177,7 @@ def __init__(self, value=long(0)):
 
     def __setattr__(self, name, value):
         # Ensure the `_value` attribute is always a long.
-        if name == '_value' and not isinstance(value, types.LongType):
+        if name == '_value' and not isinstance(value, long):
```

Comment:
Okay makes sense. In the case that we're running this code in Python 2 do you think we should still use `types`?